app/views/experiences/card/_new.html.erb

```diff
@@ -2,7 +2,7 @@
   <% if @challenge.stage_number > 1 %>
     <div class="col-xs-8 col-xs-offset-2">
       <div class='well well-lg well-share'>
-        <p class='text-bold'>Want to get involved? Contribute in <%= link_to "Stage #{@challenge.stage_number} >", challenge_active_stage_path(@challenge) %>
+        <p class='text-bold'><%= t('experiences.card._new.cta') %> <%= link_to "Stage #{@challenge.stage_number} >", challenge_active_stage_path(@challenge) %>
```

Comment:
While this works, the better form would be to pass the value that we want to be displayed here instead (similar to how we have it done in other examples).